pkg/descriptions.json

```diff
@@ -1405,6 +1405,11 @@
         ]
     },
     {
+        "Name": "System.Security.Permissions",
+        "Description": "Provides types supporting Code Access Security (CAS).",
```

Comment:
There's more than CAS in this package, so this description is overly specific to the point of being misleading.